pkg/shared/connection/api/defaultapi/default_client.go

```diff
@@ -273,3 +289,34 @@ func (a *defaultAPI) CreateOAuthTransport(accessToken string) *http.Client {
 		},
 	}
 }
+
+func (a *defaultAPI) CreateOCMConnection() (*ocmSdkClient.Connection, error) {
+	connection, err := ocmSdkClient.NewConnectionBuilder().
+		Tokens(a.AccessToken).
```

Comment:
This is fine for Prod/Stage but for testing it might be great to have the token customizable since someone using kas-installer (https://github.com/bf2fc6cc711aee1a0c2a/kas-installer) might as well have KFM deployed using a different Auth server than the one used for OCM Stage/Prod (which is always sso.redhat.com)